be2-scala/src/test/scala/ch/epfl/pop/authentication/AuthenticateSuite.scala

```diff
@@ -0,0 +1,222 @@
+package ch.epfl.pop.authentication
+
+import akka.http.scaladsl.model.StatusCodes.{Found, OK}
+import akka.http.scaladsl.model.Uri.Query
+import akka.http.scaladsl.model.{HttpRequest, HttpResponse, Uri}
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class AuthenticateSuite extends AnyFunSuite with Matchers with ScalatestRouteTest {
+
+  private val routeName = "authenticate/"
+
+  private val responseType = "id_token token"
+  private val clientID = "abc"
+  private val redirectUri = "https://wikipedia.org"
```

Comment:
nitpick: I think the domain `example.com` exists for these use case, though I don't think Wikipedia would come back at us...

Reply:
Wow I didn't know that domain actually existed for real 😳